sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TransmissionStateManager.cs

```diff
@@ -0,0 +1,144 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Internals
+{
+    internal class TransmissionStateManager
+    {
+        private const int MaxDelayInSeconds = 3600;
+
+        private const int MinDelayInSeconds = 10;
+
+        private readonly Random _random = new();
+
+        /// <summary>
+        /// Minimum time interval between failures to increment consecutive error count.
+        /// </summary>
+        private TimeSpan _minIntervalToUpdateConsecutiveErrors = TimeSpan.FromSeconds(20);
+
+        /// <summary>
+        /// Time threshold after which consecutive error count can be incremented.
+        /// </summary>
+        private DateTimeOffset _nextMinTimeToUpdateConsecutiveErrors = DateTimeOffset.MinValue;
+
+        private readonly System.Timers.Timer _backOffIntervalTimer;
```

Comment:
`Timer` is Disposable.
This class will need to clean it up.

Azure.Core has this check turned off, but I think we need to find a way to turn it back on for our projects :(

Reply:
Added